src/main/java/jarvis/logic/Event.java

```diff
@@ -0,0 +1,42 @@
+package jarvis.logic;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+
+    String start;
+    String end;
+    LocalDate dateStart;
+    LocalDate dateEnd;
```

Comment:
While not a coding standard violation, perhaps it would be best to put a `private `access modifier (and/or even `final`) before the fields? I noticed the same issue in a few other places too!